meetings/agenda.md

```diff
@@ -7,3 +7,48 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 
 ## Items to discuss
 
+### Components to be potentially moved to security-only status
+
+Proposal: these components should (probably) be clearly labeled so that maintainers know that only security
+issues are to be fixed, if they occur.
+
+ * laminas-db
+ * laminas-developer-tools
+ * laminas/laminas-oauth
+ * laminas/laminas-xml
+ * laminas/laminas-composer-autoloading
+ * laminas/laminas-auradi-config
+ * laminas/laminas-development-mode
+ * laminas/documentation
+ * laminas/laminas-barcode
+ * laminas/laminas-config-aggregator-parameters
+ * laminas/laminas-config
+ * laminas/laminas-config-aggregator-modulemanager
+ * laminas/laminas-console
+ * laminas/laminas-dom
+ * laminas/laminas-crypt
+ * laminas/laminas-mvc-console
+ * laminas/laminas-json
+ * laminas/laminas-http
+ * laminas/laminas-memory
+ * laminas/laminas-loader
+ * laminas/laminas-servicemanager-di
+ * laminas/laminas-json-server
+ * laminas/laminas-mime
+ * laminas/laminas-log
+ * laminas/laminas-math
+ * laminas/laminas-xml2json
+ * laminas/laminas-soap
+ * laminas/laminas-paginator
+ * laminas/laminas-navigation
+ * laminas/laminas-progressbar
+ * laminas/laminas-serializer
+ * laminas/laminas-tag
+ * laminas/laminas-text
+ * laminas/laminas-twitter
+ * laminas/laminas-uri
+ * laminas/laminas-mail
+ * laminas/laminas-file
+ * laminas/laminas-di
+ * laminas/laminas-skeleton-installer
```

Comment:
This is a Composer plugin used by all of our skeletons in order to prompt for optional packages. In point of fact, it will be getting updates soon so that it remains compatible with Composer v2.